packages/rolldown/build.ts

```diff
@@ -112,9 +112,6 @@ const configs = defineConfig([
                 )
                 fsExtra.copyFileSync(file, nodePath.join(copyTo, fileName))
                 console.log(colors.green('[build:done]'), `Cleaning ${file}`)
-                try {
-                  fsExtra.rmSync(file)
```

Comment:
Is removing this expected?

Reply:
This seems remove the generated binary files in `rolldown/src` after coping them to the `dist` folder.

The reason of it is make sure that if binary files is failed to be built, there aren't binary files in `rolldown/src` that are generated in last time successful build.

Reply:
Yeah, if you remove the binding here, building js-glude-code only become impossible.

Reply:
I will try to clean up the native binding if the native is failed to build in next pr..